recipes/libltc/all/conanfile.py

```diff
@@ -35,6 +35,10 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    def build_requirements(self):
+        if tools.os_info.is_windows and self.settings.compiler != "Visual Studio":
```

Comment:
```suggestion
        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
```
You'll probably need it like this (I copied from other recipes).
And you'll need to set the environment to use the cl compiler,
From automake recipe:
```python
    @contextmanager
    def _build_context(self):
        if self.settings.compiler == "Visual Studio":
            with tools.vcvars(self.settings):
                with tools.environment_append({"CC": "cl -nologo", "CXX": "cl -nologo",}):
                    yield
        else:
            yield
```